src/components/SideMenu/SideMenu.vue

```diff
@@ -2,20 +2,42 @@
   <v-card>
     <v-navigation-drawer app clipped permanent :mini-variant="mini">
       <v-list-item class="hidden-sm-and-down" data-cy="profile">
-        <v-list-item-content>
+        <v-list-item-content class="mb-0 pb-0">
           <v-list-item-avatar class="avatar" size="40%">
             <v-img src="https://randomuser.me/api/portraits/men/1.jpg"></v-img>
           </v-list-item-avatar>
           <v-list-item-title class="mt-3" data-cy="user-name">
             {{ user.firstname + " " + user.lastname }}
           </v-list-item-title>
-          <v-list-item-subtitle class="text-caption grey--text">{{
-            since
-          }}</v-list-item-subtitle>
+          <v-list-item-subtitle class="text-caption grey--text">
+            {{ since }}
+          </v-list-item-subtitle>
+          <v-container class="mb-0 pb-0 mx-0 px-0">
+            <v-row no-gutters>
+              <v-col>
+                <a 
```

Comment:
instead of anchors, I suggest using `<router-link to ="/following"> ` instead since the anchor will reload the page while the router link won't.